Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,241 @@
+       for movie in movies {
+            print(movie)
+            sleep(2)
+        }
+    
+default:
+    print("Grab some popcorn 🍿 and enjoy the movie 🎬 ")
+    sleep(2)
+}
+
+print("How satisfied where you with the selected movies according to your mood ")
+print("1...4")
+print("1 really bad it sucked, 2 boring never again, 3 Ehh it was okay could have been better  4 it was great")
+var satisfactory = readLine()
+
+
+switch satisfactory {
+case "1": // really bad it sucked
+    print("Would you like more suggestions, yes or no? ")
+    let answerChoice = readLine() ?? "".lowercased()
+    if answerChoice == "yes" {
+        print("What are you favorite genera")
+        print("A. Comedy B. romance C. Horror D. Documentaries E. action")
+        let genreChoice = readLine() ?? "".lowercased()
+        switch genreChoice {
+        case "A":
+            print("recommend movies are:")
+                let movies = ["Step B. brothers", "Wedding Crashers", "Hangover", "Stuber"]
+            
+            for movie in movies {
+                print(movie)
+                sleep(1)
+            }
+        case "B":
+            print("recommended movies are:")
+            let movies = ["Time travelers wife", "50 first dates", "The Note Book", "P.S. I love You"]
+              for movie in movies {
+                  print(movie)
+                  sleep(1)
+              }
+    
+        case "C":
+            print("recommended movies are:")
+            let movies = ["The Wrong Turn", "it", "The Nun", "Pet Sematary"]
+            for movie in movies {
+                            print(movie)
+                            sleep(1)
+                        }
+        case "D":
+            print("recommended movies are:")
+               let movies = ["What the Health", "How America was Built", "Surviving never land", "Abducted in Plain Sight"]
+               for movie in movies {
+                    print(movie)
+                    sleep(2)
+                }
+        case "E":
+            print("recommended movies are:")
+            let movies = ["John Wick", "Rambo, Last Blood", "Captain Marvel", "Avenger Endgame"]
+            for movie in movies {
+                             print(movie)
+                             sleep(2)
+                         }
+        default:
+            print()
+        }
+    } else if answerChoice == "no" {
+        print("I am sorry I was no help 😔 maybe next time !" )
+        
+    }
+    
+    
+case "2": // Boring I fell asleep
+        print("Would you like more suggestions, yes or no? ")
+        let answerChoice = readLine() ?? "".lowercased()
+        if answerChoice == "yes" {
+            print("What are you favorite genera")
+            print("A. Comedy B. romance C. Horror D. Documentaries E. action")
+            let genreChoiceTwo = readLine() ?? "".lowercased()
+            switch genreChoiceTwo {
+
+                case "A":
```

Comment:
You declared genreChoiceTwo as a lowercased which means that these choices will never occur because they are all upper cased. Make sure you are testing your code as you go to account for these little oversights.